docs/tutorials/get-started/servo-mousemover.md

```diff
@@ -151,38 +152,39 @@ The servo is now physically connected to the Raspberry Pi, but the Viam app hasn
-
 ### Board component
 
-In the **Config** tab, create a new component with the following attributes:
+Create a [board component](/components/board/):
+
+Navigate to the **Components** subtab of your robot's **Config** tab.
 
-- **Name**: `local` - you can name this whatever you want, you will reference it later.
-- **Type**: `board`
-- **Model**: `pi`
+Click **Create component** in the lower-left corner.
 
-![This is a screenshot of the Viam app on the CONFIG page. It shows that Builder is chosen as the mode. Below that in the components tab it shows Create Component and to the far left It says Name with a box and the name local typed in. The next box to the right says Type and it shows a drop down list of different types of components in Viam with board selected. To the far right it says Model with a drop down list and pi is selected. Directly to the right of Model is a Create Component button that will create the component.](/tutorials/single-component-tutorials-servo-mousemover/new-board.png)
+Select `board` for type and `pi` for model.
 
-Click **Create Component** to create the pi [board component](/components/board/) component.
+Enter `local` as the name for your board, then click **Create**.
+
+You can name the board whatever you want.
+You will reference it later.
 
 ### Servo component
 
-After the board component, create a new [servo component](/components/servo/) with the following attributes:
+Create a [servo component](/components/servo/):
+
+Click **Create component** in the lower-left corner.
 
-- **Name**: `FS90R` - we named the servo using the model number, but you can use whatever you want, you will reference it later.
-- **Type**: `servo`
-- **Model**: `pi`
+Select `servo` for type and `pi` for model.
 
-![This is a screenshot of the Viam app on the CONFIG page. It shows that Builder is chosen as the mode. Below that in the components tab it shows Create Component and to the far left It says Name with a box and the name FS90R typed in. The next box to the right says Type and it shows a drop down list of different types of components in Viam with servo selected. To the far right it says Model with a drop down list and pi is selected. Directly to the right of Model is a Create Component button that will create the component.](/tutorials/single-component-tutorials-servo-mousemover/new-servo.png)
+Enter `FS90R` as the name for your servo, then click **Create**.
 
-Click **Create Component** to create the [servo component](/components/servo/).
+We named the servo using the model number, but you can use whatever you want.
+You will reference it later.
```

Comment:
Same comment as above: recommend skipping this.

Reply:
```suggestion
```
Okay. If it weren't used in the code I'd say leave this in so users learn about naming, but it is in the code so I'll take it out to avoid confusion.

Reply:
Also, this tutorial uses two different names for the servo in different places, and then the code repo has a 3rd name, so I will make them consistent now :)

Reply:
Thank you!!